.github/workflows/deploy.yaml

```diff
@@ -0,0 +1,67 @@
+name: Build and upload to PyPI
+
+on:
+  pull_requests:
+    branches:
+      - master
```

Comment:
This will get removed once it works on CI in this PR.

For things to work you'll need to upload two tokens to github secrets (I've set them to exist within an environment named `deploy`, but you can make them global secrets too if you'd like).
1. TEST_PYPI_API_TOKEN -> the API token for the test.pypi package of alchemlyb
2. PYPI_API_TOKEN -> the API token for the pypi package of alchemlyb

I recommend having the testpypi check there (so you can tell if something went wrong before pushing to pypi), but it's not a necessity either.

Reply:
I added these two tokens to the *deploy* environment.